src/commands/blocks.ts

```
@@ -386,7 +388,7 @@ export const blocksCommand = define({
 					if (block.isGap ?? false) {
 						// Gap row
 						const gapRow = [
-							pc.gray(formatBlockTime(block, useCompactFormat)),
+							pc.gray(formatBlockTime(block, useCompactFormat, ctx.values.locale)),
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)